src/endpoint/s3/s3_bucket_policy_utils.js

```diff
@@ -136,6 +138,16 @@ async function _is_object_tag_fit(req, predicate, value) {
     dbg.log1('bucket_policy: object tag fit?', value, tag, res);
     return res;
 }
+async function _is_object_version_fit(req, predicate, value) {
+
+    if (!req) {
```

Comment:
why is this condition needed? req is already validated by _is_condition_fit

Reply:
Not needed.  lt was used for debugging.